mne/epochs.py

```diff
@@ -818,6 +869,59 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         return self._evoked_from_epoch_data(data, self.info, picks, n_events,
                                             kind, self._name)
 
+    def regress(self, on, fit_intercept=True, by=None):
```

Comment:
? Why putting this in the object?? 

I don't think that's the right way to go. We should get users to understand how to implement whatever analysis they need using standard packages, not creating wrappers in all objects. 

There already exists multiple ways to do linear regression in mne, e.g. mne.stats.linear_regression